scripts/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    "extends": "../tsconfig.json",
+    "compilerOptions": {
+        "noEmit": true,
+        "lib": ["dom", "es2016"]
```

Comment:
The parent config already has this `lib` definition.

It seems like overkill for a project this small, but there _could_ be a `baseConfig.json` that both configs extend.